recipe/meta.yaml

```diff
@@ -15,11 +15,15 @@ build:
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Could you remove `noarch: python` and add `skip: True  # [py<36]`, see https://github.com/nexB/license-expression/blob/5f4febb6df2b644706839e42ba4310de9e2dccba/setup.cfg#L48